### ISSUE TITLE
[Feature Request]: No tabbar if only one tab?

### ISSUE THREAD

**Issue by ISO8807** [mintty/mintty#1201]
Although I have enabled tabs (which can be very useful), I normally have only one tab. It looks a little odd to have a whole tabbar for this.

Would it be feasible to hide the tabbar if there is only one tab? Only on adding a second tab (i.e. window) would the tabbar become visible. On closing this second tab, the tabbar should be hidden again.

Thanks for listening.

**Comment by mintty** [comment]
Personally I'm not in favour of such auto-magic. Could be an option though.

**Comment by ISO8807** [comment]
Thanks for the reply, which is understandable.  Mintty allows the scrollbar to be turned on and off. Perhaps there could be a similar possibility to turn the tabbar on and off. Just a thought.

**Comment by ISO8807** [comment]
Out of curiosity, I did a quick hack of the source code to try this idea out. I arranged for a control character (Ctrl-B for "bar") to toggle visibility of the tabbar using win_open_tabbar or win_close_tabbar according to the result of win_tabbar_visible. That achieves what I wanted, though of course it might well not be something you want to incorporate in the main code or in the hackish way I did it.

**Comment by mintty** [comment]
Yes, I checked the same. I will add user-definable functions, maybe one function `toggle-tabbar`.

**Comment by mintty** [comment]
Finally managed to commit this...

**Comment by mintty** [comment]
Released 3.6.4.

**Comment by eabase** [comment]
How do you actually use this mysterious "tabbar"?

**Comment by ISO8807** [comment]
In case it helps, I often have several MinTTY windows open. Using the tab bar I can easily switch between these, can copy and paste between windows, etc. It's much like tabbed browsing. Perhaps your question is more about how you open and close tabs?

**Comment by mintty** [comment]
https://mintty.github.io/mintty.1.html#Virtual%20Tabs
How is this mysterious? If you point out gaps of clarity, I'll improve documentation.

**Comment by eabase** [comment]
> How is this mysterious? If you point out gaps of clarity, I'll improve documentation.

Yes, please do. 
How do you open a new tab?
How do you make tabs show, when you open the (Cygwin) mintty terminal?
